.envrc

```diff
@@ -1,3 +1,7 @@
 use asdf
 
-dotenv
\ No newline at end of file
+dotenv
+
+# used in dockercompose and dockerfile 
+export BUILDPLATFORM=$(uname -m)
```

Comment:
i tried this out locally based on this same documentation and found that the value evaluated to an empty string when I ran on my machine.

Reply:
This diff worked for me without needing to set any additional variables:
```diff
diff --git a/python_src/Dockerfile b/python_src/Dockerfile
index 06a3e1e..40d69a6 100644
--- a/python_src/Dockerfile
+++ b/python_src/Dockerfile
@@ -25,9 +25,9 @@ COPY pyproject.toml pyproject.toml

 # Tableau dependencies for arm64 cannot be resolved (since salesforce doesn't
 # support them yet). For that buildplatform build without those dependencies
-ARG BUILDPLATFORM
-RUN echo "Installing python dependencies for ${BUILDPLATFORM}."
-RUN if [ "$BUILDPLATFORM" = "arm64" ]; then \
+ARG TARGETARCH BUILDPLATFORM TARGETPLATFORM
+RUN echo "Installing python dependencies for build: ${BUILDPLATFORM} target: ${TARGETPLATFORM}"
+RUN if [ "$TARGETARCH" = "arm64" ]; then \
     poetry install --without tableau --no-interaction --no-ansi -v ;\
     else poetry install --no-interaction --no-ansi -v ;\
     fi
```